Firmware/temperature.cpp

```diff
@@ -2316,26 +2315,31 @@ static void check_temp_raw()
 #ifdef TEMP_MODEL
 namespace temp_model {
 
-void model_data::reset(uint8_t heater_pwm, uint8_t fan_pwm, float heater_temp, float ambient_temp)
+void model_data::reset(uint8_t heater_pwm _UNUSED, uint8_t fan_pwm _UNUSED,
```

Comment:
Do we need to keep the parameters if they are unused now?

Reply:
Yes. I already checked that there's no flash cost if they are unused. I want to keep the code providing the initial values if we want to change this in the future so that the model can start a few milliseconds sooner.

Reply:
ok